### ISSUE TITLE
Headphones Pause Button Doesn't Work

### ISSUE THREAD

**Issue by JayXT** [SimpleMobileTools/Simple-Music-Player#516]
Hello,
I use headphones with three buttons. Two of them for volume control in Simple Music Player work fine. The third one used for pausing (single click), next track (double click), previous track (triple click) doesn't work as expected. While the player properly goes to next/previous track with double/triple clicks, it doesn't stop the playback with a single click.
Could you please fix this?

**Comment by tibbi** [comment]
does it work in other apps?

**Comment by JayXT** [comment]
Yes, such apps as [Symphony](https://github.com/zyrouge/symphony/) and [Music](https://github.com/MaxFour/Music-Player) properly handle all three buttons and support pausing with a single click.

**Comment by JayXT** [comment]
If it helps, it's a Bluetooth headset, not the wired one.
